docs/content/concepts/cryptography/passkeys.mdx

```diff
@@ -1,60 +1,51 @@
 ---
 
 Passkey provides a secure and user-friendly alternative for submitting transactions to Sui. Built on the **WebAuthn standard**, passkey lets users authenticate and sign transactions using:
 
-- Hardware security keys, such as YubiKeys
-- Mobile devices, such as smartphones and tablets
-- Platform-based authenticators, such as Face ID and Touch ID
+- Hardware security keys, such as YubiKeys  
+- Mobile devices, such as smartphones and tablets  
+- Platform-based authenticators, such as Face ID and Touch ID  
 
-Passkey simplifies authentication by removing the need to manage seed phrases or private keys manually. Instead, they rely on-device-based authentication and cloud synchronization, allowing seamless, phishing-resistant access across multiple devices. You can use use passkey in a [multi-signature (multisig)](./transaction-auth/multisig.mdx) setup, giving you more flexibility to build secure, recoverable wallet experiences.
+Passkey simplifies authentication by removing the need to manage seed phrases or private keys manually. Instead, it relies on device-based authentication and cloud synchronization, allowing seamless, phishing-resistant access across multiple devices. You can also use passkey in a [multisig setup](./transaction-auth/multisig.mdx), which provides more flexibility to build secure and recoverable wallet experiences.
 
-By supporting the passkey signature scheme, Sui improves security and accessibility, making it easier for users to manage their accounts with hardened security. Passkey-based wallets are also tied to the origin, meaning they can’t be phished or used on a different site, which makes it a more secure authentication option.
+By supporting the passkey signature scheme, Sui improves security and accessibility, making it easier for users to manage their accounts with hardened security. Passkey-based wallets are tied to the origin, meaning they cannot be phished or used on a different site. This makes passkey a more secure authentication option.
 
-Refer to the [TypeScript SDK support](https://sdk.mystenlabs.com/typescript/cryptography/passkey) on how to add passkey support to your application. For product specification, refer to [SIP-9](https://github.com/sui-foundation/sips/blob/main/sips/sip-9.md).
+See the [TypeScript SDK documentation](https://sdk.mystenlabs.com/typescript/cryptography/passkey) to learn how to add passkey support to your application. For the product specification, see [SIP-9](https://github.com/sui-foundation/sips/blob/main/sips/sip-9.md).
 
 :::info
-Passkey support is available in beta in Sui Devnet and Testnet. The Mainnet release is yet to be scheduled.  
+Passkey support is available in beta in Sui Devnet and Testnet. The Mainnet release is not yet scheduled.  
 :::
 
 ## Benefits of using passkey
 
-**Sign transactions seamlessly**
+- **Sign transactions securely**  
```

Comment:
```suggestion
- **Sign transactions securely:**  Users can sign transactions in Sui using passkey, while the private key stays securely stored within the authenticator. This reduces the risk of key extraction attacks.
```
Might want this construction for this list if the description doesn't indent correctly. Lists are finicky like that.